packages/protocol-kit/tests/e2e/offChainSignatures.test.ts

```diff
@@ -309,7 +309,7 @@ describe('Off-chain signatures', () => {
         isSuccessful: false,
         ethGasPrice: '',
         gasUsed: 12345,
-        fee: 12345,
+        fee: '12345',
```

Comment:
`fee` is defined as an integer here: https://safe-transaction-mainnet.safe.global/
(same as `baseGas` and `safeTxGas` in lines 296-297)

Reply:
![imagen](https://user-images.githubusercontent.com/25051234/233293503-e41b8294-8100-44bc-8662-9b2ee84817b5.png)

We can't just trust swagger 🙈